plugins/woocommerce/tests/php/src/Internal/CLI/Migrator/Core/PlatformRegistryTest.php

```diff
@@ -0,0 +1,281 @@
+<?php
+/**
+ * Platform Registry Test
+ *
+ * @package Automattic\WooCommerce\Tests\Internal\CLI\Migrator\Core
+ */
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Tests\Internal\CLI\Migrator\Core;
+
+use Automattic\WooCommerce\Internal\CLI\Migrator\Core\PlatformRegistry;
+
+/**
+ * PlatformRegistryTest class.
+ */
+class PlatformRegistryTest extends \WC_Unit_Test_Case {
```

Comment:
While reviewing these tests, I wondered what would happen if we added validation for namespaces later on. No current issues, just flagging it though.

Reply:
Also, since the commands' tests are quite simple, I assume we will improve them in the future.

Reply:
>Also, since the commands' tests are quite simple, I assume we will improve them in the future.

Yep that's the plan.